packages/p2p/src/stores/order-store.js

```diff
@@ -23,10 +23,13 @@ export default class OrderStore {
     @observable error_message = '';
     @observable has_more_items_to_load = false;
     @observable is_loading = false;
+    @observable is_rating_modal_open = false;
+    @observable is_recommended = undefined;
```

Comment:
Would this be better initialized as `@bservable is_recommended = null`, since other observables in this store`order-store` initializes it as `null`

Reply:
`is_recommended` can be `null` via the response, hence `undefined` is better.